### PR TITLE
Add audit monkit stats

### PR DESCRIPTION
**Pull request by mobyvb**
What: 

Why:

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: pkg/audit/verifier.go [hunk 1]
```diff
@@ -126,6 +126,32 @@ func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe, skip map[s
 
 	successNodes := getSuccessNodes(ctx, nodes, failedNodes, offlineNodes, containedNodes)
 
+	totalInPointer := len(stripe.Segment.GetRemote().GetRemotePieces())
```
Comment (jtolds) [inline pkg/audit/verifier.go]: out of curiosity, i assume by this part in the code we know this segment isn't inline?

File: pkg/audit/verifier.go [hunk 2]
```diff
@@ -126,6 +126,32 @@ func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe, skip map[s
...
+	totalAudited := len(successNodes) + len(failedNodes) + len(offlineNodes)
```
Comment (jtolds) [inline pkg/audit/verifier.go]: nice, good job eliminating the contained nodes.

File: pkg/audit/verifier.go [hunk 3]
```diff
@@ -126,6 +126,32 @@ func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe, skip map[s
...
+	numOffline := len(offlineNodes)
+	numSuccessful := len(successNodes)
+	numFailed := len(failedNodes)
+	auditedPercentage := float64(totalAudited) / float64(totalInPointer)
+	offlinePercentage := float64(numOffline) / float64(totalAudited)
+	successfulPercentage := float64(numSuccessful) / float64(totalAudited)
+	failedPercentage := float64(numFailed) / float64(totalAudited)
```
Comment (jtolds) [inline pkg/audit/verifier.go]: division by zero risk above

File: pkg/audit/verifier.go (hunk 1 above)
Comment (mobyvb) [reply pkg/audit/verifier.go]: Yes, the cursor specifically selects a remote stripe

File: pkg/audit/verifier.go (hunk 3 above)
Comment (mobyvb) [reply pkg/audit/verifier.go]: fixed

**Pull request merged**